lib/core/feature-flags/src/lib/services/dummy-features.service.ts

```diff
@@ -0,0 +1,44 @@
+/*!
+ * @license
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { Injectable } from '@angular/core';
+import { Observable, of } from 'rxjs';
+import { IFeaturesService, FlagChangeset } from '../interfaces/features.interface';
+
+@Injectable()
+export class DummyFeaturesService implements IFeaturesService {
+    init(): Observable<FlagChangeset> {
+        return of();
+    }
+
+    isOn$(_key: string): Observable<boolean> {
```

Comment:
you can just omit the parameters instead, the linter is still going to complain on the Sonar side I think